pkg/util/namespace_filter.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"github.com/golang/glog"
+	apiv1 "k8s.io/api/core/v1"
+	"regexp"
+	"strings"
+)
+
+type NamespaceFilterConfig struct {
+	Namespace       string
+	NamespaceFilter string
+}
+
+func isNamespaceFilterEnabled(namespaceFilterConfig *NamespaceFilterConfig) bool {
+	enableFilter := false
```

Comment:
no need to have a var defined, you only need to return true or false for each case

Reply:
Done